src/model/liveselection.js

```diff
@@ -565,6 +566,11 @@ export default class LiveSelection extends Selection {
 
 			// ...look for a first character node in that range and take attributes from it.
 			for ( const item of range ) {
```

Comment:
Let's also take the chance to fix this variable name – it should be `value`. So we don't have `item.item` later on.